src/components/Form.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <>
+      <h2>Add a New Book</h2>
+      <form action="" method="">
+        <input type="text" name="title" id="title" placeholder="Book Title" />
+        <input type="text" name="author" id="author" placeholder="Book Author" />
+        <button type="submit">Add Book</button>
+      </form>
+    </>
+  );
+}
+
```

Comment:
- [ ] Since you are using this component as a add book form please remove the other empty component which is not being used [src/components/BookForm.js](https://github.com/naji-97/sec-book-store/pull/1/files#diff-c71d8d7b2b96cd9764d639f38b6bb3e9fdd6b951916d82fc50a014ad136a50ac) as the previous reviewer asked